pkg/expr/sql/parser_allow.go

```diff
@@ -138,6 +138,31 @@ func allowedNode(node sqlparser.SQLNode) (b bool) {
 	case *sqlparser.Where:
 		return
 
+	// // New safe node types
+	// case sqlparser.Values, sqlparser.ValTuple:
```

Comment:
Values cannot be used at read-time with MySQL. It's used to insert records.

I can be used at read-time with Postgres, but not MySQL